src/extension/utils.ts

```diff
@@ -559,16 +559,16 @@ export function convertEnvList(envs: string[]): Record<string, string | undefine
   )
 }
 
-export function getGithubAuthSession(createIfNone: boolean = true) {
-  return authentication.getSession(AuthenticationProviders.GitHub, ['user:email'], {
-    createIfNone,
-  })
-}
-
 export async function getPlatformAuthSession(createIfNone: boolean = true, silent?: boolean) {
```

Comment:
What's this for? Shouldn't all this be absorbed into the singleton? Seems leaky to have things call in and out of "utils"

Reply:
Good catch. Having seen this, I think we can move it into the Singleton too.

Reply:
I believe it should be named to something like this: **getOrCreateSession()**

Reply:
Sounds good